styleguide/source/_patterns/02-molecules/hub/hub-hero.md

```diff
@@ -0,0 +1,57 @@
+---
+el: ".hub-hero"
+title: "Hub Hero"
+---
+### Description
+
+A hero CTA that consists of a title, optional description, optional background image, and optional cta button. The order of elements is determined using flexbox where different from the base pattern.
+
+### Variant options
+* [As cta only](?p=molecules-hub-hero-as-cta-only)
+* [As fity fifty](?p=molecules-hub-hero-as-fifty-fifty)
+* [As fity fifty left](?p=molecules-hub-hero-as-fifty-fifty-left)
+* [As fity fifty video](?p=molecules-hub-hero-as-fifty-fifty-video)
```

Comment:
Is this supposed to be as fifty fifty? I think you're missing some f's

Reply:
Thanks. Too many "f"s